api/src/components/publicationVersion/__tests__/getAll.test.ts

```diff
@@ -67,9 +67,9 @@ describe('Get many publication versions', () => {
         });
 
         expect(getPublications.status).toEqual(200);
-        const publicationDates = getPublications.body.data.map((version) => version.publishedDate);
+        const publicationDates = getPublications.body.data.map((version) => version.publishedDate as Date);
```

Comment:
Ah yes it can, so above you don't need the :Date[] type as it's inferred (or implied?)